src/bindingHandlers/graphRenderer.ts

```diff
@@ -2919,10 +2905,14 @@ function render(graph: LogicalGraph, elementId : string, eagle : Eagle){
             normalColor = LINK_COLORS['LINK_WARNING_COLOR'];
             selectedColor = LINK_COLORS['LINK_WARNING_SELECTED_COLOR'];
         }
-
         return eagle.objectIsSelected(edge) ? selectedColor : normalColor;
     }
 
+    function getArrowType(edge: Edge, index: number) {
```

Comment:
Perhaps modify this name from 'getArrowType' to something that follows the existing pattern of function names. 'edgeGetEndMarkerUrl' or 'edgeGetArrowheadUrl'.